config/ground-truth-yaml-files/all_features_version_10_no_visit_variant_pcd.yml

```diff
@@ -2607,21 +2617,21 @@ patient:
     type: string
     search_term: venlafaxine
   FEV1_value_first:
-    categories:
+   categories:
    - biolink:PhenotypicFeature
    type: number
   FEV1_value_last:
-    categories:
+   categories:
    - biolink:PhenotypicFeature
    type: number
   FVC_value_first:
-    categories:
+   categories:
    - biolink:PhenotypicFeature
    type: number
   FVC_value_last:
-    categories:
-    - biolink:PhenotypicFeature
-    type: number
+   categories:
+   - biolink:PhenotypicFeature
+   type: number

```

Comment:
@karafecho Looks like there are spacing issues here. The old file has correct spacing here, so need to remove these spacing related changes that will cause formatting issue.